src/app/zap-templates/zcl/data-model/chip/client-monitoring-cluster.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<configurator>
+    <domain name="CHIP"/>
+
+    <struct name = "MonitoringRegistration" isFabricScoped="true">
+        <cluster code="0x0046"/>
```

Comment:
Let's use a code that is far enough from existing clusters that we don't clash with new clusters from ballot, which are much closer to being finalized. Suggest 0x1046. Alternatively, reach-out to Cam Williams to get a reserved number temporarily.

Reply:
done